tests/test_minimal/test_tools/test_expand_paths.py

```diff
@@ -98,3 +100,47 @@ def test_expand_paths(tmpdir):
     for x in string_directory_out:
         assert x in expected
     assert len(string_directory_out) == len(expected)
+
+
+def test_expand_paths_ibl(tmpdir):
+    expander = LocalPathExpander()
+
+    # set up IBL Steinmetz video file structure
+    generate_path_expander_demo_ibl(tmpdir)
+    base_directory = Path(tmpdir)
+
+    # NOTE: empty brackets used b/c for some reason NR_0021/2022-06-30/ subdir is 002/ instead of 001/
+    out = expander.expand_paths(
```

Comment:
```suggestion
    path_expansion_results = expander.expand_paths(
```

Recommend using more verbose/descriptive variable names, especially in tests